Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -146,10 +152,12 @@ namespace O3DE::ProjectManager
 
                 QMenu* newProjectMenu = new QMenu(this);
                 m_createNewProjectAction = newProjectMenu->addAction("Create New Project");
-                m_addExistingProjectAction = newProjectMenu->addAction("Add Existing Project");
+                m_addExistingProjectAction = newProjectMenu->addAction("Open Existing Project");
+                m_addRemoteProjectAction = newProjectMenu->addAction("Add Remote Project");
```

Comment:
lets hide this behind a `#define ` until the dialog actually can add a remote repo etc. 
```suggestion
#ifdef ADD_REMOTE_PROJECT_ENABLED
                m_addRemoteProjectAction = newProjectMenu->addAction("Add Remote Project");
#endif
```

Reply:
Done